wordpress-mobile/WordPress-Android
# Issue/13268 implement clear and apply actions

Partially fixes #13268

This PR adds the clear and apply action to the Activity Log Type Filter screen. 

Note: Review by commits might be easier.
Note2: Checked items are not persisted when you go back to activity log and re-open the activity type filter. That's coming in the next PR.

Merge instructions:
1. Make sure https://github.com/wordpress-mobile/WordPress-Android/pull/13470 is reviewed
2. Remove "Not ready for merge" label
3. Merge this PR

To test:
1. My site -> Activity Log
2. Click on "Activity type" chip
3. Try to select multiple items
4. Click on the Clear action and notice all items get de-selected
5. Click on the Apply action and notice the screen gets dismissed

The rest of the functionality will be tested when the filter actually work - next week.

PR submission checklist:
